project/style.css

```diff
@@ -0,0 +1,227 @@
+.login-section{
+  margin-left: 15%;
+  margin-right:15%;
+  margin-top: 35%;
+}
+
+.section-image1{
+  width:30%;
+  float:left;
+}
+
+.section-image2{
+  height:25%;
+  width:25%;
+  float:right;
+  margin-left: 52%;
+  position:absolute;
+}
+
+.right-paragraph {
+  margin-left: 25%;
+  float: right;
+  position:absolute;
+  width: 40%;
+}
+
+.left-paragraph {
+  float: left;
+  position: absolute;
+  margin-right: 25%;
+  width: 40%;
+}
+
+.header-form{
+  background-color: rgb(51, 153, 255);
+  color: rgb(255, 255, 255);
+  box-shadow: 0px 10px 30px rgb(166, 166, 166);
+  height: 20%;
+  margin-top: 0%;
+  text-align: center;
+}
+
+br { font-size: 200%; }
+
+.content-form{
+  width:30%;
+  margin-top: -30%;
+  margin-left: 20%;
+  border: 2px solid black;
+}
+
+.login-form{
+  position: absolute;
+  background-color:
+  rgb(255,255,255);
+  width: 20%;
+  height: 40%;
+}
+
+input[type=text], [type=date] {
```

Comment:
Can you describe to me what this selector is doing? Also, could you explain why `input` appears only once before the comma, but not after it?